concent_api/core/tests/test_integration_subtask_results_verify.py

```diff
@@ -468,30 +473,98 @@ def test_that_concent_should_change_subtask_state_if_verification_is_after_deadl
 
         self._test_response(
             response_2,
-            status          = 200,
-            key             = self.REQUESTOR_PRIVATE_KEY,
-            message_type    = message.concents.SubtaskResultsSettled,
-            fields          = {
+            status=200,
+            key=self.REQUESTOR_PRIVATE_KEY,
+            message_type=message.concents.SubtaskResultsSettled,
+            fields={
+                'origin': message.concents.SubtaskResultsSettled.Origin.ResultsRejected,
                 'task_to_compute': self.report_computed_task.task_to_compute,
             }
         )
 
         response_3 = self.client.post(
             reverse('core:receive_out_of_band'),
-            data         = self._create_provider_auth_message(),
-            content_type = 'application/octet-stream',
+            data=self._create_provider_auth_message(),
+            content_type='application/octet-stream',
         )
 
         self._test_response(
             response_3,
-            status          = 200,
-            key             = self.PROVIDER_PRIVATE_KEY,
-            message_type    = message.concents.SubtaskResultsSettled,
-            fields          = {
+            status=200,
+            key=self.PROVIDER_PRIVATE_KEY,
+            message_type=message.concents.SubtaskResultsSettled,
+            fields={
+                'origin': message.concents.SubtaskResultsSettled.Origin.ResultsRejected,
                 'task_to_compute': self.report_computed_task.task_to_compute,
             }
         )
 
+    def test_that_concent_should_change_subtask_state_to_failed_if_files_were_not_uploaded_on_time(self):
+        """
+        Provider -> Concent: SubtaskResultsVerify
+        Concent -> Provider: SubtaskResultsRejected
+        Concent -> Requester: SubtaskResultsRejected
+        """
+
+        # given
+        (serialized_subtask_results_verify,
+         subtask_results_verify_time_str) = self._create_serialized_subtask_results_verify()
+        subtask_results_verify_date_time = datetime.strptime(subtask_results_verify_time_str, "%Y-%m-%d %H:%M:%S")
```

Comment:
Please use `_get_timestamp_string()` function to converting datetime :)

Reply:
it's different function, but I have created a new one ;)